js/src/ui/ShortenedHash/shortenedHash.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+
+import styles from './shortenedHash.css';
+
+export default class ShortenedHash extends Component {
+  static propTypes = {
+    data: PropTypes.string.isRequired
+  }
+
+  render () {
+    const { data } = this.props;
+
+    let shortened = data.toLowerCase();
+    if (shortened[0] === '0' && shortened[1] === 'x') {
```

Comment:
any reason for not just checking `shortened.substr(0, 2) === '0x'` (as done everywhere else - consistency so we don't need to think about how xyz does it)